hawkbit-repository/hawkbit-repository-api/src/main/java/org/eclipse/hawkbit/repository/model/Action.java

```diff
@@ -142,6 +142,13 @@ default boolean isForced() {
         return ActionType.FORCED == getActionType();
     }
 
+    /**
+     * @return true when action is downloadonly, false otherwise
+     */
+    default boolean isDownloadOnly() {
```

Comment:
We need to check where in the code the comparison we call `action.getType() == DOWNLOAD_ONLY`  and replace it with this